### DISCUSSION TITLE
Welcome to expresive-tea Discussions!

### DISCUSSION THREAD

**Discussion by chrnx-dev** [Zero-OneiT/expresive-tea#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by storymx** [comment]
when are the tutorials gonna be available for public in case you may want to update the youtube video?.
